models/DayTimeFactor.js

```diff
@@ -0,0 +1,15 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+
+const factorSchema = new Schema({
+  morningfactor: { type: Number, required: true },
+  lunchfactor: { type: Number, required: true },
+  eveningfactor: { type: Number, required: true },
+});
```

Comment:
Based on your question on the discussion board I would amend your Schema like this:
```suggestion
const factorSchema = new Schema({
  _id: { type: Number },
  morningfactor: { type: Number, required: true },
  lunchfactor: { type: Number, required: true },
  eveningfactor: { type: Number, required: true },
});
```